core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,31 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner. See the administration chapter of the user manual for details on how to configure and use
+   * classloader contexts.
+   *
+   * @param classLoaderContext
+   *          name of the classloader context
+   * @throws NullPointerException
+   *           if context is null
+   * @since 1.8.0
+   */
+  void setClassLoaderContext(String classLoaderContext);
```

Comment:
Can I get the server side configuration inside a ScannerImpl or TabletServerBatchReader? I don't see how, and if I did, do we want the client to get the server side configuration?